src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -773,42 +859,85 @@ class Browser extends DashboardView {
     }
 
     if (isNewObject) {
-      // for dynamically changing required placeholder text for _User class new row object
```

Comment:
Was this removed on purpose?

Reply:
Since, dynamically changing required placeholder logic has now moved to child component `BrowserRow`.

Why did we need to move that logic down?

In this pr, we are also giving an option to edit failed cloned rows, having a state variable for keeping list of required field for that class will affect all rows that are in editing mode, i.e, if I fill `username` in first cloned edit row, it will change the placeholder for all of the rows, which is not we wanted. 

Since, each row knew which fields are filled and which are empty and also had required columns. So, I moved that logic down.

Reply:
Makes sense